orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -35,14 +35,13 @@
     </label>
     <div class="todo-qa panel panel-default" ng-if="todoQas[todo.description] && todoQas[todo.description].comment">
       <div class="panel-body">
-        <strong ng-if="todoQas[todo.description].todo === todo.id">Feedback on this project</strong>
-        <strong ng-if="todoQas[todo.description].todo !== todo.id">Feedback from previous project</strong>
-        : {{todoQas[todo.description].comment}}
+        <span>{{todoQas[todo.description].todo === todo.id?"Feedback on this project":"Feedback from previous project"}}:</span>
+        <strong>{{todoQas[todo.description].comment}}</strong>
```

Comment:
sanity-check: is there a space after the colon?

Reply:
Yes.